sycl/doc/design/Fuzzing.md

```diff
@@ -0,0 +1,104 @@
+# Fuzzing plan for intel/llvm
+
+The main product which is being developed at intel/llvm repo is a SYCL
+implementation. At high-level, it consists of two compiler and runtime and
+therefore this document will be divided into two major sections covering those
+components. Those components are essentially the only entry points through which
+a user can interact with our product.
+
+## SYCL Runtime
+
+SYCL runtime is a library which implements SYCL API, but besides that API it
```

Comment:
```suggestion
SYCL runtime (RT) is a library which implements SYCL API, but besides that API it
```

Just because we refer to it as RT later, so it this should help the reader make the shorthand association.

Reply:
We only ever use `RT` abbreviation once, to I opted in for expanding it to have a clearer language in db6aba9154a2989f998a7c77ac8e388160dd9b18